db/schema.rb

```diff
@@ -335,36 +334,4 @@
     t.datetime "updated_at",                                                 null: false
   end
 
-  create_table "users", force: :cascade do |t|
```

Comment:
This is an oversight from someones dev environment, these tables are migrations in the rad codebase and rad_consumer doesn't need to know about them.